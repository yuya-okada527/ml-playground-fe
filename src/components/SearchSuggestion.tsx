import React from "react";
import { Alert, AlertTitle } from "@material-ui/lab";
import { createStyles, makeStyles, Theme } from "@material-ui/core";

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    alert: {
      marginTop: theme.spacing(3),
      marginRight: theme.spacing(3),
    },
  })
);

/**
 * 映画検索の説明を描画する。
 */
const SearchSuggestion: React.FC = () => {
  const classes = useStyles();
  return (
    <Alert severity="info" className={classes.alert}>
      <AlertTitle>お気に入りの映画を探してみてください。</AlertTitle>
      あなたが好きな映画をもとに、おすすめの映画を紹介します。
    </Alert>
  );
};

export default SearchSuggestion;
