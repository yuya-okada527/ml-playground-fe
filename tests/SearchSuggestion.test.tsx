import React from "react";
import { shallow } from "enzyme";
import SearchSuggestion from "../src/components/SearchSuggestion";

describe("SearchSuggestion", () => {
  it("Snapshot Test", () => {
    const searchSuggestion = shallow(<SearchSuggestion />);
    expect(searchSuggestion).toMatchSnapshot();
  });
});
