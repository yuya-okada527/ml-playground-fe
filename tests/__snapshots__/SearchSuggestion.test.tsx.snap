// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchSuggestion Snapshot Test 1`] = `
<WithStyles(ForwardRef(Alert))
  className="makeStyles-alert-1"
  severity="info"
>
  <WithStyles(ForwardRef(AlertTitle))>
    <WithStyles(ForwardRef(Typography))
      variant="subtitle1"
    >
      お気に入りの映画を探してみてください。
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(AlertTitle))>
  <WithStyles(ForwardRef(Typography))
    variant="body2"
  >
    あなたが好きな映画をもとに、おすすめの映画を紹介します。
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    variant="body2"
  >
    おすすめ映画が気に入りましたら、いいねボタンのご協力お願いします。
  </WithStyles(ForwardRef(Typography))>
</WithStyles(ForwardRef(Alert))>
`;
